### PR TITLE
feat(financial-statements-inao): Update submit to handle contacts

### PR DESCRIPTION
**Pull request by fjandakornid**
# Upload contacts

## What

Upload contacts

## Why

must have contacts

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts [hunk 1]
```diff
@@ -43,6 +46,13 @@ export interface DataResponse {
   success: boolean
   message?: string
 }
+
+interface CemeteryCaretaker {
```
Comment (HelgiHH) [inline libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts:50]: should be able to import the Boardmember via `@island.is/application/templates/financial-statements-inao/types`

File: libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts [hunk 2]
```diff
@@ -214,16 +261,50 @@ export class FinancialStatementsInaoTemplateService {
...
+
+      const contacts: Contact[] = [
+        {
+          nationalId: actor.nationalId,
+          name: actorsName,
+          contactType: ContactType.Actor,
+        },
+      ]
+
+      contactsAnswer.forEach((x) => {
```
Comment (HelgiHH) [inline libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts:292]: suggest using Array.map here

**Review by HelgiHH — COMMENTED** [review]
lgtm 2 suggestions

File: libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts (hunk 1 above)
Comment (fjandakornid) [reply libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts]: used Boardmember

File: libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts (hunk 2 above)
Comment (fjandakornid) [reply libs/application/template-api-modules/src/lib/modules/templates/financial-statements-inao/financial-statements-inao.service.ts]: updated to map

**Review by HelgiHH — APPROVED** [review]

**Pull request merged**